File: android/src/main/kotlin/com/baseflow/permissionhandler/data/PermissionStatus.kt
package com.baseflow.permissionhandler.data

import com.google.gson.annotations.SerializedName

enum class PermissionStatus {
    @SerializedName("unknown")
    UNKNOWN,
    @SerializedName("denied")
    DENIED,
    @SerializedName("disabled")
    DISABLED,
    @SerializedName("granted")
    GRANTED,
}

File: ios/Classes/data/PermissionStatus.swift
//
//  PermissionStatus.swift
//  permission_handler
//
//  Created by Maurits van Beusekom on 25/07/2018.
//

import Foundation

enum PermissionStatus : String, Codable {
    case denied = "denied"
    case disabled = "disabled"
    case granted = "granted"
    case restricted = "restricted"
    case unknown = "unknown"
}
